MAUI/Button/Overview.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title: About .NET MAUI control | Syncfusion
+description: Learn here all about introduction of Syncfusion .NET MAUI Button (SfButton) control, its elements and more.
+platform: maui
+control: SfButton
+documentation: ug
+---
+
+# .NET MAUI Button (SfButton) Overview
+
+The .NET MAUI Button is a custom button control with UI customization. You can set icons, background images, and corner edge radii and customize the appearance for different visual states using the visual state manager.
```

Comment:
The .NET MAUI Button is a custom button control with UI customization. Set icons, background images, and corner edge radii and customize the appearance for different visual states using the visual state manager.

Reply:
Changes have been addressed.